detection_rules/utils.py

```diff
@@ -192,3 +192,23 @@ def clear():
 
 def clear_caches():
     _cache.clear()
+
+
+def load_rule_contents(rule_file: str, forbid_multi=False) -> list:
```

Comment:
Instead of inverted boolean flags, I vote renaming `forbid_multi` to `single_only`

Reply:
wfm 👍